chapters/publication.tex

```diff
@@ -1,96 +1,178 @@
 %------------------------------------------------
 
 \begin{fullwidth}
-Increasingly, research assistants are relied on to manage some or all
-of the publication process. This can include
-managing the choice of software,
-coordinating referencing and bibliography,
-tracking changes across various authors and versions,
-and preparing final reports or papers for release or submission.
-Modern software tools can make a lot of these processes easier.
-Unfortunately there is some learning curve,
-particularly for lead authors who have been publishing for a long time.
-This chapter suggests some tools and processes
-that can make writing and publishing in a team significantly easier.
-It will provide resources
-to judge how best to adapt your team to the tools you agree upon,
-since all teams vary in composition and technical experience.
-
+Publishing academic research today extends well beyond writing up and submitting a Word document alone.
+There are often various contributors making specialized inputs to a single output,
+a large number of iterations, versions, and revisions,
+and a wide variety of raw materials and results to be published together.
```

Comment:
I find this sentence hard to parse. Needs rewording, maybe something like:

Typically, various contributors collaborate on both code and writing, manuscripts go through many iterations and revisions, and the final package for publication includes not only the manuscript but also the code and data used to generate the results.

Reply:
d2f09af98b60a0aaa26c3077f80a3c2aacfe96aa